android/KMEA/app/src/main/java/com/tavultesoft/kmea/KMManager.java

```diff
@@ -1015,6 +1015,13 @@ public static Typeface getFontTypeface(Context context, String fontFilename) {
 
     try {
       if ((fontFilename != null) && FileUtils.hasFontExtension(fontFilename)) {
+        // Ignore .woff files if Android 7.0 / 7.1 (Issue #4896)
+        if ((Build.VERSION.SDK_INT >= Build.VERSION_CODES.N) &&
+            (Build.VERSION.SDK_INT <= Build.VERSION_CODES.N_MR1) &&
```

Comment:
https://developer.android.com/guide/topics/resources/font-resource suggests WOFF is not supported in any recent version? Have you tested on 8.0 plus?

Some Android bug reports:
* https://issuetracker.google.com/issues/62842880
* https://issuetracker.google.com/issues/37136340

Reply:
Yes, I also ran emulator with API 29 (Android 10.0) and the woff loaded fine.